etc/skydive.yml.default

```diff
@@ -417,6 +418,12 @@ k8s:
   - statefulset
   - storageclass
 
+istio:
```

Comment:
This should be in :
```
analyzer:
  topology:
    istio:
```

like for the `fabric` probe.
By the way, we should do the same for the `k8s` probe

Reply:
Lets' discuss and handle separately for both k8s and istio , once we are aligned on this I'll issue a separate PR.